api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -113,4 +115,25 @@ public void sendMessage_shouldSendMessage() throws MessageException {
 		}
 	}
 	
+	/**
+	 * @see MessageService#getMessagePreparator()
+	 */
+	@Test
+	public void getMessagePreparator_shouldGetMessagePreparator() {
+		
+		assertNotNull( ms.getMessagePreparator() );
+		
+	}
+	
+	/**
+	 * @see MessageService#setMessagePreparator(MessagePreparator)
+	 */
+	@Test
+	public void setMessagePreparator_shouldSetMessagePreparator(){
+		
+		ms.setMessagePreparator( null);
+		assertNull( ms.getMessagePreparator() );
```

Comment:
Remove the spaces. E.g. `ms.setMessagePreparator(null);` and `assertNull(ms.getMessagePreparator());`